docker-compose.yml

```diff
@@ -3,15 +3,18 @@ version: '2'
 services:
   master:
     container_name: 'citus_master'
-    image: 'citusdata/citus:6.2.3'
+    image: 'citusdata/citus:7.0.0'
     ports: ['5432:5432']
```

Comment:
I'm thinking about a better way of dealing with this (the port) to allow multiple compose clusters at once; will update tomorrow.

Reply:
Did you have a chance to look at this?